src/Frame.hpp

```diff
@@ -11,6 +12,9 @@
 
 namespace envire { namespace core
 {
+    using boost::uuids::random_generator;
+    using boost::thread_specific_ptr;
+
```

Comment:
I would avoid to use using statements in header files, since every class that includes this header will have the same namespace or type resolutions. A better option would be to use in-class typedefs.